core-primitives/node-api/api-client-types/src/lib.rs

```diff
@@ -86,6 +86,7 @@ mod api {
 	// runtime dependency here.
 	// pub use itp_types::Block;
 	pub use my_node_runtime::{Block, Runtime, UncheckedExtrinsic};
+	//pub use ita_parentchain_interface::integritee::{Block, Runtime, UncheckedExtrinsic};
```

Comment:
here we will get circular deps. to be solved in a followup step

Reply:
Yes, these type definitions should also be moved to the `ita` domain, shouldn't they?

Reply:
haha, actually, removing these lines just works ;-)
the dependency is never used. previous cleanup aroind ita_parnetchain_interface was enough